app/views/split_checkout/_payment-method.html.haml

```diff
@@ -4,6 +4,7 @@
   - available_payment_methods.each do |payment_method| 
     %div.checkout-input
       = f.radio_button :payment_method_id, payment_method.id,
+        name: "order[payments_attributes][][payment_method_id]",
```

Comment:
Seems to be non-functional right now.
I need some help here (cc @Matt-Yorkley ;))

Reply:
I also tried: 
```diff
--- a/app/controllers/split_checkout_controller.rb
+++ b/app/controllers/split_checkout_controller.rb
@@ -81,13 +81,14 @@ class SplitCheckoutController < ::BaseController
     params.require(:order).permit(
       :email, :shipping_method_id, :special_instructions,
       bill_address_attributes: PermittedAttributes::Address.attributes,
-      ship_address_attributes: PermittedAttributes::Address.attributes
+      ship_address_attributes: PermittedAttributes::Address.attributes,
+      payments_attributes: [:payment_method_id]
     )
   end
```